src/lender/coordinator.sol

```diff
@@ -15,42 +15,211 @@ pragma solidity >=0.5.15 <0.6.0;
 
 import "./ticker.sol";
 import "tinlake-auth/auth.sol";
+import "ds-test/test.sol";
 
 interface EpochTrancheLike {
     function epochUpdate(uint epochID, uint supplyFulfillment_,
         uint redeemFulfillment_, uint tokenPrice_) external;
+    function getTotalOrders(uint epochID) external returns(uint totalSupply, uint totalRedeem);
 }
 
 interface ReserveLike {
     function updateMaxCurrency(uint currencyAmount) external;
+    function balance() external returns (uint);
 }
 
-contract EpochCoordinator is Ticker, Auth {
+interface AssessorLike {
+    function calcSeniorTokenPrice(uint NAV_) external returns(uint);
+    function calcJuniorTokenPrice(uint NAV_) external returns(uint);
+    function maxReserve() external view returns(uint);
+    function calcNAV() external returns (uint);
+    function seniorDebt() external returns(uint);
+    function seniorBalance() external returns(uint);
+    function seniorRatioBounds() external returns(uint minSeniorRatio, uint maxSeniorRatio);
+}
 
+contract EpochCoordinator is Ticker, Auth, DSTest {
```

Comment:
I think it is helpful to have test logs in the contract during development.

We can remove `DSTest` in the end